packages/core/src/builder.class.ts

```diff
@@ -2639,6 +2646,8 @@ export class Builder {
     let url;
     if (isApiCallForCodegen) {
       url = `${host}/api/v1/codegen/${this.apiKey}/${keyNames}`;
+    } else if (apiEndpoint === 'query') {
+      url = `${host}/api/v3/query/${this.apiKey}/${keyNames}`;
```

Comment:
this PR re-introduces `apiEndpoint` prop, with a default of `'query'`. This fixes the rendering issue for symbols